class/features.php

```diff
@@ -0,0 +1,307 @@
+
+<?php
+
+require('../../vendor/autoload.php');
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+use GuzzleHttp\Client;
+
+//Getting secret credentials using dotenv
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
+$dotenv->safeLoad();
+/**
+ * Provides some usefull features for the checking, validating and upload files.
+ * 
+ * @method onlyAlpha().
+ *   Checks wether given str is only alphabet or not.
+ * 
+ * @method onlyDigit().
+ *   Checks wether given str is only digit or not.
+ * 
+ * @method validImage().
+ *   Checks wether given image is jpg/png and under 500kb.
+ * 
+ * @method splitMarks().
+ *   Splits marks and remove '|' and return array of marks.
+ * 
+ * @method validPhoneNo().
+ *   Checks if phone no is valid or not.
+ * 
+ * @method validMailId1().
+ *   Checks if Mail Id is valid or not using RegEx.
+ * 
+ * @method validMailBox().
+ *   Checks if Mail Id is valid or not using MailBoxLayer and Guzzle.
+ * 
+ * @method sendMail().
+ *   sends mail to given email Id.
+ * 
+ * @method getURL().
+ *   Get response body of given URL using GuzzleHTTP client request.
+ * 
+ * @property string $name
+ *  Store name off the user.
+ * 
+ * @property string $mailId
+ *  Store mailId of the user.
+ * 
+ * @property string $marks
+ *  Store marks of the user.
+ * 
+ * @property string $phoneNo
+ *  Store phoneNo of the user.
+ * 
+ * @property string $imagePath
+ *  Store path of profile-pic of the user.
+ * 
+ **/
+class features
+{
+    public $name;
+    public $mailId;
+    public $marks;
+    public $phoneNo;
+    public $imagePath;
```

Comment:
give comments for variable here. like this:
/** 
*  @var string
*     used to store the name
*/
public $name;